ui/analyse/src/treeView/common.ts

```diff
@@ -59,7 +59,8 @@ const autoScroll = throttle(200, (ctrl: AnalyseCtrl, el: HTMLElement) => {
     cont.scrollTop = ctrl.path ? 99999 : 0;
     return;
   }
-  cont.scrollTop = target.offsetTop - cont.offsetHeight / 2 + target.offsetHeight;
+  const targetOffset = target.getBoundingClientRect().y - el.getBoundingClientRect().y;
```

Comment:
why do we now require 2 calls to `getBoundingClientRect`?

Reply:
I put a note in the "Fix autoscroll offset" commit, let me copy it here:

> We've added "position: relative" to the &lt;line&gt; elements for styling purposes, but that affects the calculation of the "offsetTop" property. We can instead calculate the offset ourselves.